src/Vortice.DirectML/BufferTensorDescription.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright © Amer Koleci and Contributors.
+// Licensed under the MIT License (MIT). See LICENSE in the repository root for more information.
+
+namespace Vortice.DirectML;
```

Comment:
Can you leave space here please? Same in other files

Reply:
Done